Error from isort while running pre-commit hooks

## What happened?

I set up a new project without Docker. When I tried to run the first commit, I am getting an isort issue on users that shouldn't be there.

```
Trim Trailing Whitespace.................................................Passed
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

<app>/users/urls.py:3:1: I001 isort found an import in the wrong position
<app>/users/urls.py:4:1: I001 isort found an import in the wrong position
<app>/users/urls.py:5:1: I001 isort found an import in the wrong position
<app>/users/urls.py:6:1: I001 isort found an import in the wrong position
<app>/users/urls.py:7:1: I001 isort found an import in the wrong position
```

I have not edited the `users/urls.py` file:

```
from django.urls import path

from .views import (
    user_detail_view,
    user_redirect_view,
    user_update_view,
)

app_name = "users"
urlpatterns = [
    path("~redirect/", view=user_redirect_view, name="redirect"),
    path("~update/", view=user_update_view, name="update"),
    path("<str:username>/", view=user_detail_view, name="detail"),
]
```

My `users/views.py` is also unchanged:

```
from django.contrib import messages
from django.contrib.auth import get_user_model
from django.contrib.auth.mixins import LoginRequiredMixin
from django.urls import reverse
from django.utils.translation import ugettext_lazy as _
from django.views.generic import DetailView, RedirectView, UpdateView

User = get_user_model()


class UserDetailView(LoginRequiredMixin, DetailView):

    model = User
    slug_field = "username"
    slug_url_kwarg = "username"


user_detail_view = UserDetailView.as_view()


class UserUpdateView(LoginRequiredMixin, UpdateView):

    model = User
    fields = ["name"]

    def get_success_url(self):
        return reverse("users:detail", kwargs={"username": self.request.user.username})

    def get_object(self):
        return User.objects.get(username=self.request.user.username)

    def form_valid(self, form):
        messages.add_message(
            self.request, messages.INFO, _("Infos successfully updated")
        )
        return super().form_valid(form)


user_update_view = UserUpdateView.as_view()


class UserRedirectView(LoginRequiredMixin, RedirectView):

    permanent = False

    def get_redirect_url(self):
        return reverse("users:detail", kwargs={"username": self.request.user.username})


user_redirect_view = UserRedirectView.as_view()
```

## What should've happened instead?

The first commit should not have an error with isort.


## Steps to reproduce

[//]: # (Any or all of the following:)
Follow the [non-docker instructions](https://cookiecutterdjango.readthedocs.io/en/latest/developing-locally.html).

[//]: # (* Host system configuration: OS, Docker & friends' versions etc.) 
Mac 10.15.3, Python 3.7.5

[//]: # (* Replay file https://cookiecutter.readthedocs.io/en/latest/advanced/replay.html)

```
{
  "cookiecutter": {
    "project_name": "SupLoca",
    "project_slug": "suploca",
    "description": "Support Local Businesses",
    "author_name": "Emile Petrone",
    "domain_name": "suploca.com",
    "email": "<my email>",
    "version": "0.1.0",
    "open_source_license": "Not open source",
    "timezone": "America/New_York",
    "windows": "n",
    "use_pycharm": "n",
    "use_docker": "n",
    "postgresql_version": "11.3",
    "js_task_runner": "None",
    "cloud_provider": "AWS",
    "mail_service": "Mailgun",
    "use_drf": "n",
    "custom_bootstrap_compilation": "n",
    "use_compressor": "n",
    "use_celery": "y",
    "use_mailhog": "y",
    "use_sentry": "y",
    "use_whitenoise": "n",
    "use_heroku": "n",
    "ci_tool": "None",
    "keep_local_envs_in_vcs": "y",
    "debug": "n",
    "_template": "https://github.com/pydanny/cookiecutter-django"
  }
}
```

[//]: # (* Logs)

```
Trim Trailing Whitespace.................................................Passed
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

<app>/users/urls.py:3:1: I001 isort found an import in the wrong position
<app>/users/urls.py:4:1: I001 isort found an import in the wrong position
<app>/users/urls.py:5:1: I001 isort found an import in the wrong position
<app>/users/urls.py:6:1: I001 isort found an import in the wrong position
<app>/users/urls.py:7:1: I001 isort found an import in the wrong position
```

--- a/{{cookiecutter.project_slug}}/.pre-commit-config.yaml
+++ b/{{cookiecutter.project_slug}}/.pre-commit-config.yaml
@@ -3,18 +3,22 @@ default_stages: [commit]
 fail_fast: true
 
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: master
     hooks:
       - id: trailing-whitespace
-        files: (^|/).+\.(py|html|sh|css|js)$
+      - id: end-of-file-fixer
+      - id: check-yaml
 
--   repo: local
+  - repo: https://github.com/psf/black
+    rev: 19.10b0
+    hooks:
+      - id: black
+
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: 3.7.9
     hooks:
       - id: flake8
-        name: flake8
-        entry: flake8
-        language: python
-        types: [python]
         args: ['--config=setup.cfg']
+        additional_dependencies: [flake8-isort]
 

